src/ae/oracle.js

```diff
@@ -68,17 +67,22 @@ async function getOracleObject (oracleId) {
  * @return {Function} stopPolling - Stop polling function
  */
 function pollForQueries (oracleId, onQuery, { interval = 5000 } = {}) {
-  const knownQueryIds = new Set()
+  let from
   const checkNewQueries = async () => {
-    const queries = ((await this.api.getOracleQueriesByPubkey(oracleId)).oracleQueries || [])
-      .filter(({ id }) => !knownQueryIds.has(id))
-    queries.forEach(({ id }) => knownQueryIds.add(id))
-    if (queries.length) onQuery(queries)
+    const { oracleQueries } = await this.api.getOracleQueriesByPubkey(oracleId)
+    from = oracleQueries[oracleQueries.length - 1]?.id ?? from
+    if (oracleQueries.length) onQuery(oracleQueries)
   }
 
-  checkNewQueries()
-  const intervalId = setInterval(checkNewQueries, interval)
-  return () => clearInterval(intervalId)
+  let stopped
+  (async () => {
+    while (!stopped) { // eslint-disable-line no-unmodified-loop-condition
+      // TODO: allow to handle this error somehow
```

Comment:
so this is still work in progress?

Reply:
Nope, we need some api to handle poll exceptions, but it is out of the scope of this PR.

Probably this PR would be closed because it looks like I misunderstood the meaning of `from` parameter (see the related issue).